library/std/src/sys/sgx/abi/mem.rs

```diff
@@ -21,8 +21,15 @@ extern "C" {
 #[inline(always)]
 #[unstable(feature = "sgx_platform", issue = "56975")]
 pub fn image_base() -> u64 {
-    let base;
-    unsafe { llvm_asm!("lea IMAGE_BASE(%rip),$0":"=r"(base)) };
+    let base: u64;
+    unsafe {
+        asm!(
+            "lea IMAGE_BASE(%rip), {}",
+            lateout(reg) base,
+            // FIXME(#76738): ATT syntax is used to support LLVM 8 and 9.
```

Comment:
This doesn't require a FIXME (same for other similar comments throughout the PR)

Reply:
Fellow contributors may not know this and try to convert to intel syntax.

Reply:
Ok but `FIXME` implies this needs to be changed, which is not the case.

Reply:
I could be change in the future when #76738 resolved. 
But I could change it to a NOTE if you prefer.